### PR TITLE
Fix deployments

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Fixes an issue with deployments and the `ofn-install` CI build failing, since we merged #4142 this morning.

#### What should we test?

Deployments should work now (dev tested).

### CODE REVIEW

**Comment by Matt-Yorkley** [comment]
Deployments are now fixed :heavy_check_mark:

File: db/default/users.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def create_admin_user
     say "\nWARNING: There is already a user with the email: #{email}, so no account changes were made.  If you wish to create an additional admin user, please run rake spree_auth:admin:create again with a different email.\n\n"
   else
     admin = Spree::User.new(attributes)
-    admin.skip_confirmation!
+    admin.skip_confirmation!.skip_confirmation_notification!
```
Comment (mkllnk) [inline db/default/users.rb]: Interesting, I probably didn't see this one because I didn't have delayed job running locally.

Comment (Matt-Yorkley) [reply db/default/users.rb]: This one only occurs on a fresh deploy :point_up:

**Comment by Matt-Yorkley** [comment]
Ugh. There's a still an issue with this in the ofn-install CI build. :frowning_face:

**Pull request merged**